src/ui/WcPanel.tsx

```diff
@@ -387,10 +430,139 @@ const WcPanel = ({ records, attributes, query, actions, adapter, creatingAdapter
 
   const saveAdapterCode = function() {
     chrome.storage.local.set({ [_adapterKey]: adapterCode }, function() {
-console.log("saved changes");
+    console.log("saved changes");
     });
   }
 
+  // This pattern matches every thing after and including any one of these symbols: = ( + - * /
```

Comment:
In general code looks great, appreciate the "functional pipeline" style you're using and comments are helpful.

My main concern is that this file is getting really big and covers a lot of responsibilities. Do you see a convenient way to split out this autosuggest into its own component/file? I haven't fully thought through what that boundary would look like, but listing out what props we'd need to pass down would probably clarify some things.